### ISSUE TITLE
RecyclerView data binding support in Android

### ISSUE THREAD

**Issue by weberdl** [CodonFramework/Codon#10]
I've used the ListView (Android) data binding previously.  RecyclerView seems to be the preferred widget for displaying lists, can you add data binding support for RecyclerView?

**Comment by DVaughan** [comment]
Yes I will. I've written one already but I need to clean it up for public consumption.
